packages/trader/src/App/Containers/Modals/success-dialog.jsx

```diff
@@ -84,6 +84,7 @@ SuccessDialog.defaultProps = {
     icon_size: 'large',
     has_cancel: false,
     has_submit: true,
+    width: undefined,
```

Comment:
this has been done to avoid TS error saying that 'Type 'string' is not assignable to type 'undefined'' because it considered width to be undefined only.